src/server/controllers/effects.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Effects routes module
+ *
+ * @request  {post} /protected
+ * @request  {post} /
+ */
+
+import User from '../models/user';
+import response from '../helpers/errorHandler';
+import { NotFoundError, ClientError } from '../helpers/error';
+
+export const getEffect = async (req, res) => {
+  const { title } = req.body;
+  const { userId } = req;
+
+  try {
+    const effects = await User.findOne({ _id: userId }, 'effects');
+    const finded = effects.find(effect => effect.title === title);
```

Comment:
Funded what?
Names should be more explicit

Reply:
Done